### DISCUSSION TITLE
Does iris support CEM(Custom Entity models)?

### DISCUSSION THREAD

**Discussion by Silentgamez** [IrisShaders/Iris#1419] [Q&A] [answered]
I was wondering whether or not iris supports CEM, I searched for a while but I didn't find anyone else asking this question.

**Comment by TheNukeLore** [comment, marked as answer]
Iris is for shaders, nothing else. Other mods do might do cem however, there is one that exists but is in alpha.

**Reply by Silentgamez** [reply]
Thanks for your help.
